packages/commonwealth/client/scripts/views/pages/CreateCommunity/steps/BasicInformationStep/BasicInformationForm/BasicInformationForm.tsx

```diff
@@ -227,58 +212,6 @@ const BasicInformationForm = ({
         enableGenerativeAI
       />
 
-      <section className="header">
```

Comment:
Based on [this](https://github.com/hicommonwealth/commonwealth/issues/6119#issuecomment-1864931719) comment, I suggest we use a local flag (in the same file) to disable this code snippet instead of removing it completely.